### DISCUSSION TITLE
Enhanced Client Management

### DISCUSSION THREAD

**Discussion by this-xkit** [wg-easy/wg-easy#1310] [Ideas]
I would like to propose a feature enhancement for wg-easy. Currently, all clients are displayed in a flat list, which can become cumbersome to manage. It would be greatly beneficial if there was a way to organize clients into groups or categories. For example, you could add a person (e.g., Nikita) and then create multiple profiles under that person. In the interface, this would mean that clicking on the person would expand or show a list of their associated profiles. This would make it much easier to manage and view multiple profiles for a single client.

**Comment by eandersons** [comment]
This is very similar to #724 and is mentioned also on https://github.com/wg-easy/wg-easy/discussions/1307#discussioncomment-10405106 and #1179.

**Reply by this-xkit** [reply]
yes, but they were not commented on :) this is one way to show that the topic is in demand

**Reply by this-xkit** [reply]
`+` I just started using github, I'm not well-oriented yet

**Reply by eandersons** [reply]
> yes, but they were not commented on :) this is one way to show that the topic is in demand

To show that a topic is in demand you can simply upvote it or its comments or comment on it if you have something useful to add to the topic.
